client/my-sites/draft/index.jsx

```diff
@@ -157,8 +157,8 @@ module.exports = React.createClass( {
 		}
 
 		if ( post && post.canonical_image ) {
-			image = url.parse( post.canonical_image.uri, true );
-			imageUrl = '//' + image.hostname + image.pathname + '?w=680px';
+			image = true;
+			imageUrl = post.canonical_image.uri + '?w=680px';
```

Comment:
Do you know if there was a reason we were stripping all but the hostname and pathname previously? In other words, we're not possibly including query parameters that had not been included previously. As such, we may also want to consider using `resizeImageUrl` instead, which properly accounts for presence of other query params (naively adding `?=680px` won't always work). Else, we could prepend protocol to existing code with `image.protocol`.